zmsbackend/tests/Zmsbackend/Availability/Api/AvailabilityHistoryByScopeTest.php

```
@@ -0,0 +1,121 @@
+        $body = json_decode((string) $response->getBody(), true);
+        $this->assertIsArray($body['data']);
+        $this->assertNotEmpty($body['data']);
+        $this->assertSame('created', $body['data'][0]['action']);
+        $this->assertSame(62, array_sum($body['data'][0]['weekday']));
+        $this->assertSame(2, $body['data'][0]['weekday']['monday']);
+        $this->assertSame(0, $body['data'][0]['weekday']['saturday']);
+        $this->assertSame('Terminkunden', $body['data'][0]['type']);
+        $this->assertSame('15min', $body['data'][0]['slotTime']);
+        $this->assertSame('3/3', $body['data'][0]['workstations']);
+        $this->assertSame('0-5', $body['data'][0]['bookable']);
+        $this->assertSame('Neue Öffnungszeit', $body['data'][0]['description']);
```

PHPStan (2.2.7)

[error] Method BO\Zmsbackend\Tests\Availability\Api\AvailabilityHistoryByScopeTest::testRendering() should return Psr\Http\Message\ResponseInterface but return statement is missing.


(return.missing)